core/src/core/transaction.rs

```diff
@@ -501,7 +501,7 @@ impl TransactionBody {
 		kernels: Vec<TxKernel>,
 		verify_sorted: bool,
 	) -> Result<TransactionBody, Error> {
-		let body = TransactionBody {
+		let mut body = TransactionBody {
```

Comment:
:+1:
Originally wrote this to avoid making the body `mut` unless we _needed_ it to be (i.e. if we needed to sort it).
But it definitely reads better like this.

Reply:
I was torn on this one -- technically doesn't need to be a part of this PR, but I felt it was a bit less noisy.

generally agree though that only mut binding where absolutely needed is better.